### PR TITLE
dev-python/pipenv: bump version to 2020.11.4

### PR DESCRIPTION
**Pull request by oz123**
Also, improve dependency declarations.

Closes: https://bugs.gentoo.org/748138
Package-Manager: Portage-3.0.8, Repoman-3.0.2
Signed-off-by: Oz Tiram <oz.tiram@gmail.com>

### CODE REVIEW

File: dev-python/pipenv/pipenv-2020.11.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/certifi[${PYTHON_USEDEP}]
+	>=dev-python/pip-18.0[${PYTHON_USEDEP}]
```
Comment (monsieurp) [inline dev-python/pipenv/pipenv-2020.11.4.ebuild:25]: This version of pip does not exist in the tree anymore. `dev-python/pip[${PYTHON_USEDEP}]` should suffice here.

File: dev-python/pipenv/pipenv-2020.11.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	>=dev-python/virtualenv-20.0.35[${PYTHON_USEDEP}]
+	>=dev-python/virtualenv-clone-0.2.5[${PYTHON_USEDEP}]
```
Comment (monsieurp) [inline dev-python/pipenv/pipenv-2020.11.4.ebuild:27]: Ditto.

File: dev-python/pipenv/pipenv-2020.11.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+		<dev-python/pytest-5[${PYTHON_USEDEP}]
+		dev-python/pytz[${PYTHON_USEDEP}]
+	)"
+
+src_prepare() {
+	# remove vendored version of PyYAML that is backported to Python2
+	# this should be removed when upstream removes support for Python2
+	rm -vR "${S}/${PN}/patched/yaml2/" || die
+	# disable running of unittests in parallel with xdist
+	sed -i 's/addopts = -ra -n auto/;&/g' setup.cfg
```
Comment (monsieurp) [inline dev-python/pipenv/pipenv-2020.11.4.ebuild:44]: `|| die`

File: dev-python/pipenv/pipenv-2020.11.4.ebuild [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	sed -i 's/plugins = xdist/;&/g' setup.cfg
```
Comment (monsieurp) [inline dev-python/pipenv/pipenv-2020.11.4.ebuild:45]: `|| die`

**Review by monsieurp — CHANGES_REQUESTED** [review]

File: dev-python/pipenv/pipenv-2020.11.4.ebuild (hunk 2 above)
Comment (oz123) [reply dev-python/pipenv/pipenv-2020.11.4.ebuild]: Done.

File: dev-python/pipenv/pipenv-2020.11.4.ebuild (hunk 4 above)
Comment (oz123) [reply dev-python/pipenv/pipenv-2020.11.4.ebuild]: Done.

File: dev-python/pipenv/pipenv-2020.11.4.ebuild (hunk 1 above)
Comment (oz123) [reply dev-python/pipenv/pipenv-2020.11.4.ebuild]: Done.

File: dev-python/pipenv/pipenv-2020.11.4.ebuild (hunk 3 above)
Comment (oz123) [reply dev-python/pipenv/pipenv-2020.11.4.ebuild]: Done.

**Comment by oz123** [comment]
@monsieurp thank you for the thorough review. I addressed all the comments.

**Review by monsieurp — APPROVED** [review]

**Pull request closed without merging**